src/plans/augmented_lagrangian_plan.jl

```diff
@@ -103,70 +106,27 @@ function set_manopt_parameter!(alg::AugmentedLagrangianGrad, ::Val{:λ}, λ)
 end
 
 # default, that is especially when the `grad_g` and `grad_h` are functions.
-function (LG::AugmentedLagrangianGrad)(M::AbstractManifold, X, p)
-    gp = get_inequality_constraints(M, LG.co, p)
-    hp = get_equality_constraints(M, LG.co, p)
+function (LG::AugmentedLagrangianGrad)(
+    M::AbstractManifold, X, p, range=NestedPowerRepresentation()
```

Comment:
Isn't `range` already assumed by `LG.co`? What is this useful for?

Reply:
It's usually passed down from there, yes; then we do not have to assume a default. But to stay non-breaking (and to have it nicer to use) we should.

Reply:
Or to be more precise: this function needs a range to return the “the right thing” the user asks for. Setting a default here (and in many other places) makes It easier to use (if the user does not want to think about it he gets the – old – default).
That's why the range always defaults to something in the last positional argument. I even added `j=:` as a default wherever reasonable.

Reply:
I see, OK.